app/Providers/LumenAuraConfig.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace Ushahidi\App\Providers;
+
+use Aura\Di\Container;
+use Aura\Di\ContainerConfig;
+
+class LumenAuraConfig extends ContainerConfig
+{
+    public function define(Container $di)
+    {
+        $this->configureAuraServices($di);
+        $this->injectAuraConfig($di);
+    }
+
+    public function modify(Container $di)
```

Comment:
IDN. From the docs

> Modify service objects after the Container is locked.

However... we can actually remove it because it's defined in the parent class. I thought I had to keep it for the interface.